web/src/emoji_picker.ts

```diff
@@ -864,21 +915,25 @@ function register_click_handlers() {
         );
     });
 
-    $("body").on("click", "#set-user-status-modal #selected_emoji .status-emoji-wrapper", (e) => {
-        e.preventDefault();
-        e.stopPropagation();
-        toggle_emoji_popover(e.currentTarget, undefined, {placement: "bottom"});
-        if (is_open()) {
-            // Because the emoji picker gets drawn on top of the user
-            // status modal, we need this hack to make clicking outside
-            // the emoji picker only close the emoji picker, and not the
-            // whole user status modal.
-            $(".app, .header, .modal__overlay, #set-user-status-modal").css(
-                "pointer-events",
-                "none",
-            );
-        }
-    });
+    $("body").on(
+        "click",
+        "#set-user-status-modal #selected_emoji .status-emoji-wrapper",
+        function (this: HTMLElement, e) {
```

Comment:
This linter was warning `e.currentTarget` is type `any`. I changed the function to be a **non arrow** and replaced `e.currentTarget ` with `this`.